ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,71 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
+- [ ] Guidance on when to use this item
+- [ ] Guidance on when not to use this item or consider alternatives
+- [ ] Accessibility considerations
+
+## Validate HTML
+If you are using Google Chrome or Firefox consider getting the [axe plugin](https://www.deque.com/axe) for validation.
+**What to check:**
+- If testing on W3.org
+  - [ ] Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+- If testing with axe plugin
+  - [ ] Run the ax plugin on the component or pattern. You should not receive any violations. Some best practices errors such as `Means to skip repeated blocks` of content, or `No main section` can be ignored as they do not apply.
+
```

Comment:
☝️ Added some info about axe plugins

Reply:
I'd like to revise the sentence about best practices a bit to clarify why these warnings don't apply. Maybe something like:

"Best practices warnings like `Means to skip repeated blocks of content` or `No main section` are page-level warnings and generally be ignored when developing new components."

Reply:
Thanks updated 👍